sqlmesh/core/schema_loader.py

```diff
@@ -62,10 +64,13 @@ def _get_columns(table: str) -> t.Optional[t.Dict[str, t.Any]]:
                 logger.warning(f"Unable to get schema for '{table}': '{e}'.")
                 return None
 
+        gateway_part = {"gateway": gateway} if gateway else {}
```

Comment:
Provided that `gateway` is optional in the arguments, should we just always set it and avoid dict manipulations?

Reply:
Do you mean like write `gateway: None` to the YAML?

The goal of this was to omit the key entirely if the gateway wasn't specified